plugins/qeta-react/src/components/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -150,29 +150,42 @@ export const MarkdownEditor = (props: MarkdownEditorProps) => {
   const catalogApi = useApi(catalogApiRef);
   const config = useApi(configApiRef);
 
-  const loadUserSuggestions = async (text: string) => {
+  const loadEntitySuggestions = async (text: string) => {
     if (!text) {
       return NO_SUGGESTIONS;
     }
-    const users = await catalogApi.queryEntities({
-      filter: { kind: 'User' },
-      limit: 5,
-      fullTextFilter: {
-        term: text,
-        fields: [
-          'metadata.name',
-          'metadata.title',
-          'spec.profile.displayName',
-          'spec.profile.email',
-        ],
-      },
-    });
+    const [users, groups] = await Promise.all([
+        catalogApi.queryEntities({
+        filter: { kind: 'User' },
```

Comment:
This could probably be done with `kind: ['user', 'group']` with a single query. Maybe later also support configuration which kinds can be mentioned but that's a bit more complex